src/sensitivities/linalg/diagonal.jl

```diff
@@ -3,14 +3,14 @@ export diagm, Diagonal
 
 const ∇ScalarDiag = Diagonal{<:∇Scalar}
 
-@explicit_intercepts diagm Tuple{∇AbstractVector}
+@explicit_intercepts diagm Tuple{∇AbstractVector} 
 function ∇(
     ::typeof(diagm),
     ::Type{Arg{1}},
     p,
     Y::∇AbstractMatrix,
     Ȳ::∇AbstractMatrix,
-    x::∇AbstractVector
+    x::∇AbstractVector,
```

Comment:
Is it convention to have a comma after a last element, like this? At the moment, we apparently do both, see e.g. `src/sensitivity.jl:81` and `src/sensitivity.jl:118`. We should probably make this consistent.

Reply:
Invenia's style guide says yes. We can bring Nabla in line with this over time though.